photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunction.scala

```diff
@@ -59,6 +69,30 @@ class GameEstimatorEvaluationFunction(
     (evaluations.head._2, model)
   }
 
+  /**
+   * Vectorize a [[Seq]] of prior observations.
+   *
+   * @param observations Prior observations in estimator output form
+   * @return Prior observations as tuples of (vector representation of the original estimator output, evaluated value)
+   */
+  override def convertObservations(observations: Seq[GameResult]): Seq[(DenseVector[Double], Double)] = {
+
+    val slope = 1.0 / range.end - range.start
```

Comment:
I believe this should be:

```scala
val slope = 1.0 / (range.end - range.start)
```

Also: I'm assuming the purpose for splitting it this way is to avoid recomputing slope/intercept for each observation?

Reply:
Yes, thank you for pointing this out.
And yes, because range is fixed for all.